Description: debianization: examples: neutralization
 It appeared that a couple of examples failed on some architectures:
 for now, in order to step forward, these very examples are neutralized.
 This is not a downgrade as this is was present in the previous package
 but not taken into account. This is a temporary and Debian centric patch.
Origin: vendor, Debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2018-06-08

diff --git a/examples/Makefile.am b/examples/Makefile.am
index c66c5df..300b5ac 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -10,7 +10,7 @@ LDADD = $(top_builddir)/lib/libblitz.la
 check_PROGRAMS = arrayx cartesian cast complex-test convolve \
 deriv fixed io iter matmult nested numinquire outer \
 polymorph prettyprint rand2 random reduce simple \
-slicing stencil2 storage tiny \
+slicing storage tiny \
 useret where whitt
 # cfd curldiv diff erf indirect pauli pick qcd rangexpr stencil3 stencil stencilet transform
 
@@ -47,7 +47,7 @@ random_SOURCES = random.cpp
 reduce_SOURCES = reduce.cpp
 simple_SOURCES = simple.cpp
 slicing_SOURCES = slicing.cpp
-stencil2_SOURCES = stencil2.cpp
+#stencil2_SOURCES = stencil2.cpp
 #stencil3_SOURCES = stencil3.cpp
 #stencil_SOURCES = stencil.cpp
 #stencilet_SOURCES = stencilet.cpp
